plugins/techdocs/src/config.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AppConfig, ConfigReader } from '@backstage/config';
+
+export function getDocumentationStorageUrlFromAppConfig(): string | undefined {
+  const appConfigs = (process.env.APP_CONFIG as unknown) as
```

Comment:
This won't give you the correct config, you want to grab it via an API or react context instead.